explorer/explorerroutes.go

```diff
@@ -994,3 +994,59 @@ func (exp *explorerUI) AgendasPage(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusOK)
 	io.WriteString(w, str)
 }
+
+// StatsPage is the page handler for the "/stats" path
+func (exp *explorerUI) StatsPage(w http.ResponseWriter, r *http.Request) {
+	var stats StatsInfo
+	exp.MempoolData.RLock()
+	stats.TotalSupply = exp.ExtraInfo.CoinSupply
+	stats.UltimateSupply = txhelpers.UltimateSubsidy(exp.ChainParams)
+	stats.TotalSupplyPercentage = float64(stats.TotalSupply) / float64(stats.UltimateSupply) * 100
+	stats.DevFunds = exp.ExtraInfo.DevFund
+	stats.DevAddress = exp.ExtraInfo.DevAddress
+	stats.POWDiff = exp.ExtraInfo.Difficulty
+	stats.BlockReward = exp.blockData.BlockSubsidy(int64(exp.blockData.GetHeight()), 5).Total
+	stats.NextBlockReward = exp.ExtraInfo.NBlockSubsidy.Total
```

Comment:
Instead of all these `stats.<fieldname> = ` assignments, please simultaneously declare and define, as in:

```go
stats := StatsInfo{
	TotalSupply: exp.ExtraInfo.CoinSupply,
	....
}
```